parser/internal/bytestring/bytestring.go

```diff
@@ -9,6 +9,11 @@ import (
 
 const MAX_COMPACT_SIZE uint64 = 0x02000000
 
+const OP_0 uint8 = 0x00
```

Comment:
It would be better style to declare all of these in a block like:
```
const (
    OP_0 uint8 = 0x00
    ....
)```

Reply:
Also, [idiomatically](https://golang.org/doc/effective_go.html#mixed-caps) these names should be camel case instead of capitals and underscores. They probably also don't need to be exported, so: `op0`, `op1Negate`, and so on. It may be an exception here if we're trying to match exact C++ code. I seem to have thought that previously, but really the above const _should_ have been called `maxCompactSize`.